examples/sound/Sound.cpp

```diff
@@ -87,8 +88,11 @@ int main()
     // Play a sound
     playSound();
 
-    // Play a music
-    playMusic();
+    // Play music from an ogg file
+    playMusic("orchestral.ogg");
+    
+    // Play music from an flac file
+    playMusic("song.flac");
```

Comment:
Where does it come from? Is it free? And 1 MB, it's a lot, when I released SFML 2.0 I remember that I replaced all the resources to reduce the total size of SDKs. And it's also better if the music is not too long for this example.